baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/RedundantModifier.java

```diff
@@ -45,38 +45,65 @@
         severity = BugPattern.SeverityLevel.ERROR,
```

Comment:
i'm a tiny bit concerned that having this at `ERROR` level might introduce unnecessary friction when people just want to run things like `./gradlew test` locally.  We're just detecting redundant code, not actually 'wrong' in anywa way... given that excavator can come around and auto-fix, what do you think about putting this as suggestion?

Reply:
Agreed, I initially used error because this in some part replaces a checkstyle check which we verify premerge.
Given that robots can fix this for us, and it's not a correctness issue, I don't mind moving it to a suggestion, and allowing minor regressions until the robots sweep through.